tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -14,13 +14,14 @@
 from c7n.manager import ResourceManager
 from c7n.query import sources
 from tencentcloud.common.exception.tencent_cloud_sdk_exception import TencentCloudSDKException
-
+from linq import Query
```

Comment:
preference to use jmespath here or manual code versus picking up a new dependency for a single usage.

Reply:
Thanks, we have adjusted